src/__test__/calculate.test.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
```

Comment:
You did a great job testing some functionalities and scenarios of the `calculate.js` module, but you are missing the following functionalities and scenarios: `AC`, `.`, `+/-`, `User pressed an operation after pressing '='`, `User pressed an operation button and there is an existing operation`. Please add the necessary code to cover all the functionalities & scenarios.

Reply:
Thanks for your review. We have a test case for pressing the button `AC`, please see the file [userInteractions.test.js](https://github.com/Hombre2014/math-magicians/blob/test/src/__test__/userInteractions.test.js). We will add the rest of the test scenarios you have mentioned in our next pull request.